### PR TITLE
searchPage: Create search page module

### PR DESCRIPTION
**Pull request by jennyvallon**
Search page module will render search results upon user input. 

Fixes: CNNETANTWO-88 (former CNNETANTWO-72) http://tickets.turner.com/browse/CNNETANTWO-88
Reviewed By: Jenny Vallon <jenny.vallon@turner.com>, Sean Joseph <sean.joseph@turner.com>, Alex Sover <alex.sover@turner.com>

Test: 
Starting from the home page, enter criteria into search in nav.
Fun fact: "poop" returns search results. 
For a failing case insert an string like: lreghlsjhgslkdfjhgls
By default 10 cases will be returned. More may be out of the scope of the ticket as pagination will have to be implemented.

On search success:
image

When no results are returned:
image

### CODE REVIEW

File: src/pages/searchResults/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        const results = get(this, 'props.fetchSearchResultsPayload.result', []);
+        console.log(results);
+        return (
+            <div>
+                <Helmet>
+                    <title>CNN Apple News Tools Factors Front End-Search Results</title>
+                </Helmet>
+                <PaperPage>
+                    <ContentList contentList={results}/>
+                    {/* <p>No results returned</p> */}
```
Comment (khowen) [inline src/pages/searchResults/index.js]: The search is working for me when results, but when no results, I don't see the "No results returned" message. Should this be uncommented to show in that use case?

File: src/redux/selectors/searchSelectors.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import get from 'lodash/get';
+
+export function searchState(state) {
+    return get(state, 'search.data', {}); // search keyname is mapped to keyname in root reducer
```
Comment (khowen) [inline src/redux/selectors/searchSelectors.js]: Can comments be line above so can keep it below 80 characters per line?

File: src/redux/reducers/searchReducer.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        return {...state,
+            fetching: false,
+            fetched:true,
+            message:null,
+            data:action.payload
+        }
+
+    case types.FETCH_SEARCH_RESULTS_FAILURE:
+        return {...state,
+            fetching: false,
```
Comment (zoitec) [inline src/redux/reducers/searchReducer.js]: Not a show stopper:  But just wondering why you are not setting  **fetched:true** here?

File: src/pages/searchResults/index.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import React, { Component }  from 'react';
```
Comment (jennyvallon) [inline src/pages/searchResults/index.js]: agreed

File: src/pages/searchResults/index.js (hunk 1 above)
Comment (jennyvallon) [inline src/pages/searchResults/index.js]: If you check now. it will show that message.

File: src/redux/selectors/searchSelectors.js (hunk 2 above)
Comment (jennyvallon) [inline src/redux/selectors/searchSelectors.js]: yes ma'am will change in the next PR. Alex has taken this one over.

**Pull request merged**